lib/oc_add_gh_team_to_nsp.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -euo pipefail
+
+usage() {
+    cat << EOF
+$0 [OPTIONS]
+
+Retrieves the list of users from a GitHub team 
+and adds a RoleBinding for each user in the provided namespace sets
+
+Maintainer: Matthieu Foucault <matthieu@button.is>
+
+Options:
+
+  --token
+    The GitHub personal access token used to access the GitHub team members list.
+    The token needs read access to orgs. Go to https://github.com/settings/tokens to create a token.
+  -o, --org
+    The GitHub organisation in which the team is located. Defaults to bcgov.
+  -t, --team
+    The GitHub team
+  -pp, --project-prefixes
+    The comma-separated project prefixes where the team members will be added. e.g. "abc123,456qwe"
+  -ps, --project-suffixes
+    The comma-separated project suffixes where the team members will be added. Defaults to "tools,dev,test,prod"
```

Comment:
I would maybe add just a documentation on how the script will build the namespace set? Since the dash could be a source of confusion
something like:
`-pp "abc123" -ps "tools,dev"` will affect the following namespaces:
- abc123-tools
- abc123-dev